package.json

```diff
@@ -28,49 +28,43 @@
   },
   "resolutions": {
     "@types/express": "^4.17.7",
```

Comment:
Is this necessary even with the updated libs ?

Reply:
probably not, I'll check